.jshintrc

```diff
@@ -7,6 +7,7 @@
     "beforeEach": true,
     "afterEach": true,
     "describe": true,
-    "it": true
+    "it": true,
+    "Promise": true
```

Comment:
Promises aren't natively supported in older versions of Node - please use `var Promise = require('bluebird');` in any files that need it.